recipe/data.go

```diff
@@ -1233,3 +1233,204 @@ var Trade = Response{
 		},
 	},
 }
+
+// WellbeingLocalAuthority is wellbeing data with a time codelist of "yyyy-yy"
+var WellbeingLocalAuthority = Response{
+	ID:     "e8238bad-e248-4008-92cf-d29c087741b7",
+	Alias:  "WellbeingLocalAuthority",
+	Format: "v4",
+	InputFiles: []file{
+		{"wellbeingExcel"},
+	},
+	OutputInstances: []instance{
+		{
+			DatasetID: "wellbeing-local-authority",
+			Editions:  []string{"time-series"},
+			Title:     "Well-being by local authority",
+			CodeLists: []CodeList{
+				{
+					ID:          "yyyy-yy",
+					Name:        "time",
+					HRef:        "http://localhost:22400/code-lists/yyyy-yy",
+					IsHierarchy: false,
+				}, {
+					ID:          "admin-geography",
+					Name:        "geography",
+					HRef:        "http://localhost:22400/code-lists/admin-geography",
+					IsHierarchy: true,
+				}, {
+					ID:          "wellbeing-estimate",
+					Name:        "estimate",
+					HRef:        "http://localhost:22400/code-lists/wellbeing-estimate",
+					IsHierarchy: false,
+				}, {
+					ID:          "wellbeing-measureofwellbeing",
+					Name:        "allmeasuresofwellbeing",
+					HRef:        "http://localhost:22400/code-lists/wellbeing-measureofwellbeing",
+					IsHierarchy: false,
+				},
+			},
+		},
+	},
+}
+
+	
+// Overseas travel and tourism recipe
+var OverseasTravelTourism = Response{
+	ID:     "35b7fe99-b7db-4237-9af0-f8c2c6c0935c",
+	Alias:  "Overseas Travel and Tourism",
+	Format: "v4",
+	InputFiles: []file{
+		{"Overseas v4"},
+	},
+	OutputInstances: []instance{
+		{
+			DatasetID: "overseas-travel-and-tourism",
+			Editions:  []string{"time-series"},
+			Title:     "Overseas Travel and Tourism",
+			CodeLists: []CodeList{
+				{
+					ID:          "calendar-years",
+					Name:        "time",
+					HRef:        "http://localhost:22400/code-lists/calendar-years",
+					IsHierarchy: false,
+				}, {
+					ID:          "calendar-month",
+					Name:        "month",
+					HRef:        "http://localhost:22400/code-lists/calendar-month",
+					IsHierarchy: false,
+				}, {
+					ID:          "ott_regions",
+					Name:        "regions",
+					HRef:        "http://localhost:22400/code-lists/ott_regions",
+					IsHierarchy: false,
+				}, {
+					ID:          "ott_ukos",
+					Name:        "ukos",
```

Comment:
does ukos mean anything to people familiar with this data? should we think about naming the code list more descriptively?